webapp/src/features/event/basic/EventDetails.tsx

```diff
@@ -1,14 +1,40 @@
 import { useEventOperations } from "../outletContext";
 import EventBasicDetailsForm from "./EventBasicDetailsForm";
 import "react-datepicker/dist/react-datepicker.css";
+import { Button, Card, CardBody, Form } from "react-bootstrap";
+import { getSignupsPostedMessage } from "../../../util/messageWriters";
 
 const EventDetails = () => {
 
     const [eventScratchpad, proposeEventChange] = useEventOperations();
+    const message = getSignupsPostedMessage(eventScratchpad);
 
     return <>
-        <h2 className="display-6">Event Setup</h2>
-        <EventBasicDetailsForm event={eventScratchpad} onEventChange={proposeEventChange} />
+        <Card>
+            <CardBody>
+                <h2 className="display-6">Event Setup</h2>
+                <EventBasicDetailsForm event={eventScratchpad} onEventChange={proposeEventChange} />
+            </CardBody>
+        </Card>
+        <Card className="mt-4">
+            <CardBody>
+                <div>
+                    <h3 className="mb-3">Signup Sheet</h3>
+                    <p>At the start of the week (usually Monday), unlock the S4 signup spreadsheet and post a link to it in <a target="_blank"
+                        rel="noopener noreferrer"
+                        href="https://discord.com/channels/1004489038159413248/1204320477732929566">
+                            #scheduling
+                    </a> in Discord.</p>
+                    <p>
+                        Here's a sample message to get you started. <strong>Be sure to add a ping to <span style={{color: "#9B59B6"}}>@Saltare Musica Hospite</span> at the end!</strong>
+                    </p>
+                    <div className="d-grid gap-2">
+                        <Form.Control as="textarea" value={message} rows={8} readOnly className="has-fixed-size" />
+                        <Button color={"primary"} className="mt-3 mb-2" onClick={() => { navigator.clipboard.writeText(message); }}>Copy Text</Button>
```

Comment:
At the moment this is semi-hardcoded text that isn't actually reflected in the Event object anywhere. It also is not editable at all by the user - they have to copy it into discord first to make any changes to the message text. 

Do you think it would be better to represent this in the data somewhere, in a similar way that discord announcement messages are done - or maybe even just have it directly editable here?

This applies to the text boxes in both EventDetails.tsx (the "post signup sheet" section) and EventVerifyDJs.tsx .

Reply:
Does pinging work if you include the "@saltare..." in the text here?

Reply:
> Does pinging work if you include the "@saltare..." in the text here?

Nope, unfortunately not

Reply:
....Update it totally works and it just doesn't color-highlight it before you hit send, so I assumed it didn't. lmao